examples/golang/README.md

```diff
@@ -0,0 +1,41 @@
+# AWS Golang Client Example
+
+## Introduction
+
+This directory contains examples on how to use [AWS SDK for Go V1](https://github.com/aws/aws-sdk-go) to experience the RadosGW extensions with the S3 API.
+
+## Feature
+
+We support the following features in our example.
```

Comment:
you also create the bucket here

Reply:
Thanks very much for the comments! I will add the 'create a bucket' feature in the README file and a createBucket() function in the append_object.go file.